exec_manager/dao/job_dao.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
```

Comment:
A DAO is ideally an abstract class or an interface such as:
````
from abc import ABC, abstractmethod

class JobDAO(ABC):
    
    @abstractmethod
    def create(exec_profile, workflow, inputs, ...) -> str:
        ...

    @abstractmethod
    def get(job_id) -> Job:
         ...
```

Und eine spezielle implementation, z.B. für SQL:
```
class SqlJobDAO(ABC):
    
    def __init__(db_url: str):
          self._engine = create_engine(db_url)
          ...

    @abstractmethod
    def create(exec_profile, workflow, inputs, ...) -> str:
        # an actual sql query here

    @abstractmethod
    def get(job_id) -> Job:
         # another sql query here
```